logic/system.js

```diff
@@ -29,8 +29,8 @@ async function getHiddenServiceUrl() {
 async function getElectrumConnectionDetails() {
     try {
         const address = await diskLogic.readElectrumHiddenService();
-        const port = constants.ELECTRUM_PORT;
-        const connectionString = `${address}:${constants.ELECTRUM_PORT}:t`;
+        const port = String(constants.ELECTRUM_PORT);
```

Comment:
Why do you need to cast to string here?

Reply:
Not necessary, but with this all the object elements are strings.